common/lib/xmodule/xmodule/tabs.py

```diff
@@ -1016,6 +396,22 @@ def need_name(dictionary, raise_error=True):
     return key_checker(['name'])(dictionary, raise_error)
 
 
+# Link Functions
+def link_reverse_func(reverse_name):
+    """
+    Returns a function that takes in a course and reverse_url_func,
+    and calls the reverse_url_func with the given reverse_name and course' ID.
+    """
+    return lambda course, reverse_url_func: reverse_url_func(reverse_name, args=[course.id.to_deprecated_string()])
+
+
+def link_value_func(value):
```

Comment:
This is only used by lms/djangoapps/courseware/tabs.py so we should move it there.